package.json

```diff
@@ -86,6 +86,6 @@
     "yargs": "^17.7.2"
   },
   "peerDependencies": {
-    "sass": "^1.62.1"
+    "sass": "^1.58.1"
```

Comment:
@simeonoff @damyanpetev 

Previous value was `^1.53.0` any value higher than that will be affecting some version. For example in App Builder we still use `angular@15.1.1` which depends on `sass"@1.57.1`.

Currently this doesn't affect us because our package-lock has locked igniteui-theming at "1.4.1", but it can affect other users.

So, I think we should ask what Angular versions we want to support and set the corresponding sass dependency starting at that version.
Otherwise, I guess we should consider a major version bump when requiring newer sass version.

Reply:
Aye, really depends on the version range the `igniteui-angular` package expresses depending on this one. Unfortunately...
it includes major version bumps:
https://github.com/IgniteUI/igniteui-angular/blob/04225178f07b4abc5ae2443c190cd90dfd771ce8/projects/igniteui-angular/package.json#L79
So you're absolutely right, this could still end up creeping into projects using 15.1 and getting moved into a nested node_modules because of that.

Ideally, the peer dependency _**should not**_ be the latest and greatest, but rather the minimum version supported. Feel free to use newer versions for perf improvements and have the CI verify the minimal version still builds or w/e.

Reply:
Then again, it's also super clunky trying to sync with Angular's versions, considering this package is not for that alone. Could also add yet another load path just in case for everyone, but that keeps feeling like passing the tooling shortcomings to the users instead. IMO, sass module handling - either not conceived to work with redistributable package libraries in the npm ecosystem or it's just not there yet.